src/components/interaction-designer/block-types/MobilePrimitives_MessageBlock.vue

```diff
@@ -13,6 +13,10 @@
         :block="block" />
       <block-name-editor :block="block" />
 
+      <block-output-branching-config
```

Comment:
Got this console error when expand to see configuration panel
<img width="918" alt="Screen Shot 2021-07-22 at 10 56 44 AM" src="https://user-images.githubusercontent.com/68745918/126678629-fcd07d6f-f2ff-47ea-af9e-2c74e231e60f.png">

Reply:
Then output branching fire error each time we click on them

Reply:
@bzabos , no more an issue, thnx for quick fixing